packages/site-client/src/components/wco-element-page.ts

```diff
@@ -84,26 +120,46 @@ export class WCOElementPage extends WCOPage {
     const fields = declaration.members?.filter((m) => m.kind === 'field');
     const methods = declaration.members?.filter((m) => m.kind === 'method');
 
-    return html`
-      <h1>${packageName}/${elementName}</h1>
-      <h3>${declaration.summary}</h3>
-
-      <p>${unsafeHTML(elementDescriptionHtml)}</p>
-
-      <h2>Usage</h2>
-      <pre><code>
-import '${getElementImportSpecifier(packageName, ceExportRef)}';
-     </code></pre>
+    const summary =
```

Comment:
We need some kind of word and possibly markdown aware prefixing for this string when we derive a summary from a description. Need to add TODO and an issue.